src/storage/mvcc/txn.rs

```diff
@@ -27,7 +27,7 @@ pub struct GcInfo {
 
 /// Generate the Write record that should be written that means to to perform a specified rollback
 /// operation.
-fn make_rollback(
+pub(in crate::storage) fn make_rollback(
```

Comment:
just make these `pub(crate)`, I don't think it is too important to narrow the scope that much